core/src/main/java/com/bankbazaar/tripmanager/manager/UsersManager.java

```diff
@@ -0,0 +1,84 @@
+package com.bankbazaar.tripmanager.manager;
+
+import com.bankbazaar.tripmanager.model.Users;
+import com.bankbazaar.tripmanager.repository.UsersRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Optional;
+
+public class UsersManager {
```

Comment:
Need a login method here. Take an email and password as input and check if its correct. Return the user details if login is success else return null

Reply:
i have implemented user login using spring boot starter security.